system-test/client-side-metrics.ts

```diff
@@ -205,7 +209,7 @@ function checkSingleRowCall(
  *   Cloud Monitoring.
  */
 async function checkForPublishedMetrics(projectId: string) {
-  const monitoringClient = new MetricServiceClient(); // Correct instantiation
+  const monitoringClient = new MetricServiceClient({projectId}); // Correct instantiation
```

Comment:
The test only passes when the secondary projectId is passed into the metricServiceClient.